compiler/ir/backend.common/src/org/jetbrains/kotlin/backend/common/lower/optimizations/FoldConstantLowering.kt

```diff
@@ -107,16 +107,16 @@ class FoldConstantLowering(
     private fun buildIrConstant(startOffset: Int, endOffset: Int, type: IrType, v: Any?): IrConst<*> {
         val constType = type.makeNotNull()
```

Comment:
This is now used only once below in `constType.isString()`, and probably can be replaced with `type.isStringClassType()`

Reply:
Thanks. Done.